examples/api-tests/src/file-search.spec.js

```diff
@@ -55,6 +55,60 @@ describe('file-search', function () {
 
         });
 
+        describe('#filterAndRange', () => {
+
+            it('should return the default when not searching', () => {
+                const filterAndRange = quickFileOpenService['filterAndRange'];
+                assert.equal(filterAndRange, quickFileOpenService['filterAndRangeDefault']);
+            });
+
+        });
```

Comment:
We can add an additional test to confirm the `filterAndRange` is updated after performing a search:

```ts
it('should update when searching', () => {
    quickFileOpenService['onType']('foo:2:1', () => { }); // perform a mock search.
    const filterAndRange = quickFileOpenService['filterAndRange'];
    assert.equal(filterAndRange.filter, 'foo');
    assert.deepEqual(filterAndRange.range, { start: { line: 1, character: 0 }, end: { line: 1, character: 0 } });
});
```